grails-app/services/org/pih/warehouse/order/OrderService.groovy

```diff
@@ -372,6 +374,11 @@ class OrderService {
     }
 
     Order saveOrder(Order order) {
+        if (order.destinationParty?.id != sessionManager.getCurrentLocation().organizationId) {
```

Comment:
Won't it be better to have a convenience method `getCurrentOrganizationId` method in the session manager to avoid breaking the law of demeter?